### PR TITLE
CONTRAST-35964 added info related to group matching feature

### PR DESCRIPTION
**Pull request by ubermitchy**
Documenting the group matching feature recently added.
The wording might be a little awkward O_O. Suggestions are welcome.

### CODE REVIEW

**Comment by kaitlynmortimer** [comment]
@ubermitchy Does this apply for SaaS and EOP?

**Comment by ubermitchy** [comment]
> @ubermitchy Does this apply for SaaS and EOP?

Yes! It applies to both.

File: content/installation/setup/auth/SSO.md [hunk 1]
```diff
@@ -99,6 +99,43 @@ Once connected, you can return to the **SSO** tab to view and edit your settings
...
 
+## Implement Group Membership
+
+For EOP and SaaS accounts with SSO enabled, a system administrator can automatically add users to organizational groups in Contrast according to group membership configured in the IdP. To set this up for your organization, you must update your SAML configuration in your IdP provider and your **SSO** settings in Contrast UI. 
+
+### Configuration 
+
+#### SAML 
+
+Contrast expects group information to be sent by the IdP as a multi-value attribute `contrast_group` in your SAML configuration.
```
Comment (sourabhkatti) [inline content/installation/setup/auth/SSO.md]: @ubermitchy I thought the field name was `contrast_groups` - as in plural. Can we double check this?

File: content/installation/setup/auth/SSO.md [hunk 2]
```diff
@@ -99,6 +99,43 @@ Once connected, you can return to the **SSO** tab to view and edit your settings
...
+        <saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                                xsi:type="xs:string"
+                                >GROUP2</saml2:AttributeValue>
+        ...
+    </saml2:Attribute>
+</saml2:AttributeStatement>
+```
+
+> **Note:** The attribute values listed under `contrast_group` must exactly match an existing group name. Contrast won't create new groups based on the values listed under this attribute.
```
Comment (sourabhkatti) [inline content/installation/setup/auth/SSO.md]: I think `contrast_group` is plural - `contrast_groups`

File: content/installation/setup/auth/SSO.md (hunk 1 above)
Comment (sourabhkatti) [inline content/installation/setup/auth/SSO.md]: multi-value as in comma-separated? can we be a little more clear here?

**Comment by kaitlynmortimer** [comment]
@sourabhkatti The step to enable user provisioning is included in the EOP and SaaS sections above this one.

File: content/installation/setup/auth/SSO.md (hunk 1 above)
Comment (ubermitchy) [reply content/installation/setup/auth/SSO.md]: There's an example of the element below (see the AttributeStatement)

**Comment by sourabhkatti** [comment]
Oh perfect, my mistake I missed the Enable user provisioning steps already in the docs. I think we're good to go

**Pull request merged**